crates/telemetry/src/lib.rs

```diff
@@ -16,6 +19,11 @@ mod traces;
 pub use propagation::extract_trace_context;
 pub use propagation::inject_trace_context;
 
+const OTEL_ERROR_INTERVAL: Duration = Duration::from_millis(1500);
+thread_local! {
+    static LAST_OTEL_ERROR: Cell<Instant> = Cell::new(Instant::now() - OTEL_ERROR_INTERVAL);
+}
```

Comment:
Does `thread_local!` let you inline this into the function below as with `static FIRST_OTEL_ERROR`?

Reply:
(Yes, it does)

Reply:
Very sneaky, will do.